browser/src/ConstraintTable/MitochondrialConstraintTable.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import {
+  MitochondrialGeneConstraint,
+  ProteinMitochondrialGeneConstraint,
+  RNAMitochondrialGeneConstraint,
+} from '../GenePage/GenePage'
+import { BaseTable } from '@gnomad/ui'
+
+const isProteinMitochondrialGeneConstraint = (
+  constraint: MitochondrialGeneConstraint
+): constraint is ProteinMitochondrialGeneConstraint =>
+  Object.prototype.hasOwnProperty.call(constraint, 'exp_lof')
+
+const ConstraintRow = ({
+  category,
+  expected,
+  observed,
+  oe,
+  oeLower,
+  oeUpper,
+}: {
+  category: string
+  expected: number
+  observed: number
+  oe: number
+  oeLower: number
+  oeUpper: number
+}) => (
+  <tr>
+    <th scope="row">{category}</th>
+    <td>{expected.toFixed(1)}</td>
+    <td>{observed.toFixed(1)}</td>
+    <td>
+      {oe.toFixed(2)} ({oeLower.toFixed(2)}-{oeUpper.toFixed(2)})
```

Comment:
I wonder if we could label this with `o/e = ...` to be more in line with the non-mito gene constraint table, or since there's only one 'contraint metric' in this cell, just label the column 'Observed/Expected'. It took me a few seconds to understand what that cell was showing without the context of some sort of label.

There are also a few other bells and whistles (that visual indication of o/e) that could be added to be more in line, but this may be getting into more of a question for Matt or KC type territory.

Mito gene constraint table
- ![Screenshot 2024-05-02 at 09 51 57](https://github.com/broadinstitute/gnomad-browser/assets/59549713/ea2419b1-0fe0-45bf-9204-da75a754c2aa)


Non-mito gene constraint table
- ![Screenshot 2024-05-02 at 09 51 52](https://github.com/broadinstitute/gnomad-browser/assets/59549713/39fbdd5f-b2e2-4286-83cd-2231ec589af7)